packages/styleguide/src/components/Button/Button.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { oneOf } from 'prop-types';
+import styled from 'styled-components';
+import cx from 'classnames';
+
+import { rem } from './../../style/utils';
+
+const CLASS_ROOT = 'btn';
+
+const propTypes = {
+  variant: oneOf(['preview', 'code-toggle'])
+};
+
+/* Styles have to be on the top because of Tag and TagTitle */
+const StyledButton = styled.button`
+  display: inline-block;
+  min-height: ${rem(24)};
+  margin-bottom: ${props => props.theme.spaces.small};
+  padding: 0 ${props => props.theme.spaces.small};
+  vertical-align: middle;
+  text-align: center;
+  cursor: pointer;
+  appearance: none;
+  border: 0px solid transparent;
+  background: transparent;
+  font-family: ${props => props.theme.fontFamily};
+  font-weight: ${props => props.theme.fontWeights.bold};
+  font-size: ${props => props.theme.fontSizes.small};
+  color: ${props => props.theme.colors.black};
+
+  &--preview {
+    padding-right: 0;
+    text-transform: uppercase;
+  }
+
+  &--code-toggle {
```

Comment:
Tento variant by som mozno aj nechal, ale pomenoval by som ho podla toho ako ten variant vyzera, nie podla toho na co sa teraz pouziva. To my nepride univerzalne. 

Mozno zistis ze to nema byt ani univerzalne, lebo sa to neda nikde inde pouzit a tym padom je to zbytocny variant.

Reply:
tento variant vyzerá ako tab. :D je to fake tab :D

Reply:
ale nemôžem to nazvať tab.